stereo_image_proc/src/nodes/stereo_image_proc.cpp

```diff
@@ -106,6 +106,10 @@ int main(int argc, char **argv)
   if (private_nh.getParam("queue_size", queue_size))
     shared_params["queue_size"] = queue_size;
 
+  int downsampling_factor;
+  if (private_nh.getParam("downsampling_factor", downsampling_factor))
+    shared_params["downsampling_factor"] = downsampling_factor;
```

Comment:
Could you please add this to `stereo_image_proc/launch/stereo_image_proc.launch` with the default value of 1?

Reply:
Yes, done